optax/losses/_segmentation.py

```diff
@@ -236,6 +277,83 @@ def multiclass_generalized_dice_loss(
   )
 
 
+def dice_plus_ce_loss(
```

Comment:
In the spirit of modularity, maybe this can be removed? Can the user add the dice and ce losses themselves? Do you see an advantage to exposing this as a single function?